### PR TITLE
March 2018 release notes

### PR DESCRIPTION
**Pull request by smlance**

### CODE REVIEW

File: source/blog/2018-03-06-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+---
+
+Our March releases have arrived!
+
+[Finagle 18.3.0][finagle], [Finatra 18.3.0][finatra], [Scrooge 18.3.0][scrooge], [TwitterServer 18.3.0][twitterserver], and [Util 18.3.0][util].
+
+### [Finagle](https://github.com/twitter/finagle/) ###
+
+New Features
+--------------
```
Comment (yufangong) [inline source/blog/2018-03-06-release-notes.md]: I think the '_' need to align with the subtitle for markdown, actually we may not need the underscores here.

File: source/blog/2018-03-06-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: 🌷 March 2018 Release Notes - Version 18.3.0
+published: true
+post_author:
+  display_name: Stefan Lance
+  twitter: @stefan_lance
```
Comment (yufangong) [inline source/blog/2018-03-06-release-notes.md]: LOL, I made the same change and then changed back. https://github.com/finagle/finagle.github.io/pull/51

File: source/blog/2018-03-06-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+  * finagle-http: The HTTP client will not attempt to retry nacked requests with streaming
+    bodies since it is likely that at least part of the body was already consumed and therefore
+    it isn't safe to retry. [a787955b](https://github.com/twitter/finagle/commit/a787955bc277a9f6928c29a05bb5ee7ddc3185df)
+
+Breaking API Changes
+
+  * finagle-base-http: Removed `c.t.f.http.Cookie.comment_`, `c.t.f.http.Cookie.comment_=`,
+    `c.t.f.http.Cookie.commentUrl_`, and `c.t.f.http.Cookie.commentUrl_=`. `comment` and `commentUrl`
+    per RFC-6265. NOT FOUND: D137538
```
Comment (yufangong) [inline source/blog/2018-03-06-release-notes.md]: whoops, i think this is https://github.com/twitter/finagle/commit/13a6bd2c75339397eb8dbada68dcf0a85d6529aa.

Comment (nepthar) [reply source/blog/2018-03-06-release-notes.md]: Nice catch! Maybe I should update the script to crash if it can't find one!

Comment (yufangong) [reply source/blog/2018-03-06-release-notes.md]: I think that would be great!

Comment (smlance) [reply source/blog/2018-03-06-release-notes.md]: Agreed. Great eyes, @yufangong 😃

**Comment by taylorleese** [comment]
🚢 🇮🇹

**Pull request merged**